docs/pages/machine-id/getting-started.mdx

```diff
@@ -271,9 +267,23 @@ node-name  127.0.0.1:3022 arch=x86_64,group=api-servers
 To use Machine ID with the OpenSSH integration, run the following command to
 connect to `node-name` within cluster `example.com`.
 
+```code
+$ ssh -F /opt/machine-id/ssh_config root@node-name.example.com
+```
+
+In addition to the `ssh` client you can use `tsh`. Replace the `--proxy` parameter
+with your proxy address. 
+
+<ScopedBlock scope={["oss","enterprise"]}>
+```code
+$ tsh ssh --proxy=teleport.example.com -i /opt/machine-id/identity root@node-name
```

Comment:
Nice work on showing `tsh` as well!

Reply:
Applied @zmb3 's edit on showing the auth server.  Please both approve if you're good to go on the udpates.